From 85cf09ec3b4d9b5cf309e1e481e5d6c253eea169 Mon Sep 17 00:00:00 2001
From: Orlando Vazquez <ovazquez@gmail.com>
Date: Wed, 7 Sep 2016 15:08:22 -0700
Subject: [PATCH] CNAPI-668 CNAPI test uses `moray.client._emitResults`,
 causing test to fail, however it's unneeded since the mocked `findObjects`
 emits records itself Reviewed by: Trent Mick <trentm@gmail.com> Approved by:
 Trent Mick <trentm@gmail.com>

---
 test/model/test-model-platform.js | 4 ----
 test/model/test-model-server.js   | 4 ----
 2 files changed, 8 deletions(-)

diff --git a/test/model/test-model-platform.js b/test/model/test-model-platform.js
index 484c235..7137311 100644
--- a/test/model/test-model-platform.js
+++ b/test/model/test-model-platform.js
@@ -83,10 +83,6 @@ function testListPlatformsAll(test) {
                   '20150819T020110Z': { latest: true }});
             test.done();
         });
-
-        setTimeout(function () {
-            moray.client._emitResults(expSearchResults);
-        }, 100);
     });
 }
 
diff --git a/test/model/test-model-server.js b/test/model/test-model-server.js
index eae0bf1..7d47a6d 100644
--- a/test/model/test-model-server.js
+++ b/test/model/test-model-server.js
@@ -166,8 +166,6 @@ function testListServersByUuids(test) {
 
             test.done();
         });
-
-        moray.client._emitResults(expSearchResults);
     });
 }
 
@@ -228,8 +226,6 @@ function testListServersSetup(test) {
                 'moray history should match');
             test.done();
         });
-
-        moray.client._emitResults(expSearchResults);
     });
 }
 
-- 
2.21.0

